css/excercise_one/excercise_one.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Excercise one CSS</title>
+  <link rel="stylesheet" href="excercise_one.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="main-container">
+    <nav>
+      <a>
+        <img src="../../img/Logo_with_name.png" alt="Firm's logo" class="logo">
+      </a>
+      <div class="nav-link">
+        <a href="#" class="small-light-text">Características</a>
+        <a href="#" class="small-light-text">Precio</a>
+        <a href="#" class="small-light-text">Contacto</a>
+      </div>
+      <div class="nav-register">
+        <a href="#" class="small-light-text">Ingresar</a>
+        <a href="#" class="sign-up link-border">Registrarse gratis</a>
+      </div>
+    </nav>
+    <div class="home">
+      <div class="main-page">
+        <div class = "main-page-corner">
+          <img src="../../img/group 2384.png" alt="Logo" class="corner-logo">
+          <p class="corner-title">Empresas</p>
+        </div>
+        <div>
+          <h2 class="main-phrase">Generá acciones<br> que <span class="main-phrase-second-part">sorprendan a<br> tu equipo y clientes</span></h2>
+        </div>
+        <div class="home-buttons">
+          <button class="register-button register-text-button">
+            REGISTRATE ACÁ
+          </button>
+          <button class="video-button video-text-button">
+            VER VIDEO TUTORIA
+          </button>
+        </div>
+        <div class="users-corner">
+          <div class="user-photo-background">
+            <img src="../../img/Union.png" alt="User" class="user-photo">
+          </div>
+          <p class="desired-users">Para personas &gt;</p>
+        </div>
+        <div class="planIT">
```

Comment:
classes should not be camelCase but with '-' as separators ( you could use 'planit' or 'plan-it'